envydis/gm107.c

```diff
@@ -1595,6 +1595,43 @@ static struct insn tabf0a8_1[] = {
 	{ 0, 0, OOPS },
 };
 
+static struct insn tabea80_0[] = {
+	{ 0x0000000010000000ull, 0x0000000010000000ull, N("ba") },
+	{ 0x0000000000000000ull, 0x0000000000000000ull },
+	{ 0, 0, OOPS },
+};
+
+static struct insn tabea80_1[] = {
+	{ 0x0000000000000000ull, 0x0000007000000000ull, N("u32") },
+	{ 0x0000001000000000ull, 0x0000007000000000ull, N("s32") },
+	{ 0x0000002000000000ull, 0x0000007000000000ull, N("u64") },
+	{ 0x0000003000000000ull, 0x0000007000000000ull, N("f32.ftz.rn") },
+	{ 0x0000005000000000ull, 0x0000007000000000ull, N("s64") },
+	{ 0x0000006000000000ull, 0x0000007000000000ull, N("sd32") },
+	{ 0x0000007000000000ull, 0x0000007000000000ull, N("sd64") },
+	{ 0, 0, OOPS },
+};
+
+static struct insn tabea80_2[] = {
```

Comment:
Looks like this is incomplete. e.g.

        /*0008*/               @P0 SUATOM.D.1D.ADD.S32.IGN R0, [R0], R0, 0x0;      /* 0xea08000000000000 */
        /*0008*/               @P0 SUATOM.D.1D.ADD.F32.FTZ.RN.IGN R0, [R0], R0, 0x0;      /* 0xea18000000000000 */
        /*0008*/               @P0 SUATOM.D.1D.ADD.S64.IGN R0, [R0], R0, 0x0;      /* 0xea28000000000000 */
        /*0008*/               @P0 SUATOM.D.1D.ADD.SD64.IGN R0, [R0], R0, 0x0;      /* 0xea38000000000000 */

Same values as above, just shifted over by a few bits.